test/common/README.md

```diff
@@ -453,6 +454,14 @@ Reads a Domain String and returns a Buffer containing the domain.
 Takes in a parsed Object and writes its fields to a DNS packet as a Buffer
 object.
 
+## Duplex pair Module
+
+The `common/duplexpair` module exports a single function `MakeDuplexPair`,
```

Comment:
`MakeDuplexPair()`?

Reply:
Another nit: is it OK to name a not constructor function with a first upper case letter?

Reply:
Fwiw, I would have written it as a constructor/ES6 class if that wouldn’t have made the code more complicated, and that’s why it’s uppercase. I am happy to change it to whatever you prefer :)

Reply:
If this will not make users try to use it with `new`, I am OK with any name)

Reply:
I would prefer this not be capitalized.